RELEASE_NOTES.md

```diff
@@ -9,6 +9,11 @@
 - The minimum supported version of `protobuf` was bumped to 5.28.1 and to 1.66.1 for `grpcio`. Make sure to update your dependencies accordingly.
 - The `AddComponentBoundsRequest.validity_duration` field was replaced by `AddComponentBoundsRequest.request_lifetime` to match the fields in `SetComponentPowerActiveRequest` and `SetComponentPowerReactiveRequest`. The default value was also changed from 5 to 60 seconds.
 - The `AddComponentBoundsResponse.ts` field was renamed to `AddComponentBoundsResponse.valid_until` for extra clarity and also to match the fields in `SetComponentPowerActiveResponse` and `SetComponentPowerReactiveResponse`.
+- The version of `frequenz-api-common` has been updated to v0.7.0. Please refer to the [release notes](https://github.com/frequenz-floss/frequenz-api-common/releases/tag/v0.7.0) and update your code accordingly.
```

Comment:
Commits still seem a bit screwed. "Bump frequenz-repo-config to 0.13.5" is advancing the `frequenz-api-common` submodule instead of the commit bumping `frequenz-api-common` to v0.7.0.

Reply:
Yeah, I mean the **git submodule**

![image](https://github.com/user-attachments/assets/3bfbceb1-bbac-4f17-96b8-e0ae185c5b2a)